solutions/temp_conversion.py

```diff
@@ -0,0 +1,50 @@
+def convert_temperature(celsius):
```

Comment:
Update the function signature to include type annotations:

```suggestion
def convert_temperature(celsius: float) -> dict[str, float]:


```

Reply:
Ok that is really helpful